tenets/codelingo/code-review-comments/receiver-name-short-reflect-identity/test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+type TestStruct struct{}
+
+func (t TestStruct) Method1() {
+	fmt.Println("Hello from Method1")
+}
+
+func (ts TestStruct) Method2() {
```

Comment:
I would've thought that "ts" is acceptable because it is the initials of Test Struct, but it actually passes because the substring "ts" is present in the string "teststruct". What is the thinking behind accepting "ts" as a valid receiver name?

Reply:
I believe that is enough for a two letter string to reflect the identity. The user can choose which two letters which might not necessarily be the first two letters or the initials of a two-word camel case type.